### ISSUE TITLE
ONBUILD MOUNT does not seem to work

### ISSUE THREAD

**Issue by bobef** [grammarly/rocker#28]
Hi. I don't know if this is supposed to work with rocker, I know you can achieve the same with rocker in other ways, like INCLUDE.

I have a base docker image which I use as a template. It has almost nothing in it except some ONBUILD instructions for the derived images. I tried to move it to rocker now and I added ONBUILD MOUNT. And I get an error:

```
Trigger 9, mount /apps/providerkit:/providerkit
Step 0 : MOUNT 
2015/09/27 13:11:45 Failed to process json stream error: Unknown instruction: MOUNT
```

My base Rockerfile looks like this (excerpt). On top of the ONBUILD I'm trying to also make the ONBUILD COPY instruction include a variable, so it can be customized later in the derived image :). The idea here is to copy different SDK version in the derived images.

```
ONBUILD MOUNT {{.project_providerkit}}:/providerkit
ONBUILD COPY /providerkit/{{"{{"}}.sdk_version{{"}}"}} /app/lib/providerkit
```

**Comment by ybogdanov** [comment]
Hi @bobef,

Yes, rocker does not support `ONBUILD`. I should have warned about it in the readme. Since you seem to be a power user already :) I encourage you to try [the new rocker](https://github.com/grammarly/rocker/tree/v1) that was completely rewritten and is client-side based, no fallback to `docker build`.

Also, can't use templates in the ONBUILD instructions to achieve what you want. Because rocker parses template as plain text before processing the commands, without any knowledge of the Rockerfile structure. That's why the ONBUILD body will be saved *rendered*, and it cannot be processed by the inherited Rockerfile.

**Comment by ybogdanov** [comment]
Resolved in v1.0.0 release!
